app/assets/stylesheets/app/typography.scss

```diff
@@ -22,17 +22,36 @@ h1, h2, h3, h4, h5 {
   color: black;
 }
 
-h1 {
+h1 { /* "page title" */
+  font-size: 3.167em;
   margin-top: 0.5em;
-  border-bottom: 1px solid #ddd;
 }
 
-h3 {
+.welcome { /* "welcome text" */
+  font-size: 2em;
+  color: #A3A3A3;
+  line-height: 1.25em;
+}
+
+h2 { /* "instrument title" */
+  font-size: 2.417em;
+}
+
+h3, th { /* "static title" */
   margin-bottom: 0;
+  font-weight: bold;
+  font-size: 1em;
```

Comment:
I think it might be worth going around the "manage" section for various places we use `h3`s. There are a few places where this now doesn't seem to have enough visual weight. There are already `h1` (facility name) and `h2` (product name) on some pages, so `h3` does seem to be the correct semantically.

Examples:
![screen shot 2018-08-10 at 7 17 43 pm](https://user-images.githubusercontent.com/1099111/43986009-1ab383be-9cd2-11e8-8efa-6a9cd07b9fd7.png)
![screen shot 2018-08-10 at 7 17 26 pm](https://user-images.githubusercontent.com/1099111/43986010-1ac28db4-9cd2-11e8-9ae0-3b7f690b4a23.png)

Reply:
![screen shot 2018-08-10 at 7 21 04 pm](https://user-images.githubusercontent.com/1099111/43986062-b0637d2e-9cd2-11e8-969e-da1d230b57b0.png)